docs/examples/tf/pets_training/requirements.sh

```diff
@@ -1,9 +1,2 @@
 # installing required packages
-pip install contextlib2
-git clone https://github.com/tensorflow/models.git
-cd models/research
-protoc object_detection/protos/*.proto --python_out=.
-cp object_detection/packages/tf2/setup.py .
-python -m pip install .
-pip install tensorflow==2.11.0
-cd ../../
+pip install contextlib2 pillow
```

Comment:
please add this single line in the download_preprocess_...sh and get rid of this file